core/src/main/java/com/scalar/db/transaction/consensuscommit/ConsensusCommitProvider.java

```diff
@@ -14,7 +18,9 @@ public String getName() {
 
   @Override
   public DistributedTransactionManager createDistributedTransactionManager(DatabaseConfig config) {
-    return new ConsensusCommitManager(config);
+    return new ActiveTransactionManagedDistributedTransactionManager(
```

Comment:
Wrapped the Consensus Commit transaction manager instance using decorator pattern.